cfn-resources/cloud-backup-snapshot/.rpdk-config

```diff
@@ -1,12 +1,13 @@
 {
     "artifact_type": "RESOURCE",
-    "typeName": "MongoDB::Atlas::CloudProviderSnapshots",
+    "typeName": "MongoDB::Atlas::BackupSnapshot",
```

Comment:
Update name to "MongoDB::Atlas::CloudBackupSnapshot" here and throughout the resource

Reply:
This issue is fixed